src/main/java/employeetracker/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,43 @@
+package employeetracker.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import employeetracker.model.Model;
+
+
+/**
+ * Sort and lists all persons in address book by name contains any of the argument keywords.
```

Comment:
Perhaps this should be `list all employees in Employee Tracker` instead to make it more consistent with our naming conventions?

Reply:
done